### ISSUE TITLE
Can't start MMEX after upgrade to b1d17dbf3..4b382eeab

### ISSUE THREAD

**Issue by vomikan** [moneymanagerex/moneymanagerex#6129]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/6836805/5d340b7f-1a31-4e99-bf9c-fe6a4b5ab1f9)

![image](https://github.com/moneymanagerex/moneymanagerex/assets/6836805/bb36294b-b15b-417b-bb40-b72f3f88f3f9)

![image](https://github.com/moneymanagerex/moneymanagerex/assets/6836805/6b0021bd-8bc8-43f5-bd0c-ad978ba580de)

![image](https://github.com/moneymanagerex/moneymanagerex/assets/6836805/b8692d2f-4680-4bdc-8994-81353be74236)

**Comment by n-stein** [comment]
Is a positional parameter required on the %d format specifier here?

https://github.com/moneymanagerex/moneymanagerex/blob/master/src/images_list.cpp#L459

**Comment by n-stein** [comment]
> Need a way to upgrade the five(?) existing themes by installing the new icon. I assume you do this already.

The standard MMEX themes are repackaged from source every time the app is built, so as long as the icon is included in https://github.com/moneymanagerex/themes and the themes submodule is properly updated in https://github.com/moneymanagerex/moneymanagerex there will not be any icons missing from the bundled themes. Submodule update was already done in the initial Tags commit, otherwise the beta builds would throw an error and force-close when you tried to run them.

> But what about any user defined themes? Are they going to throw an error if the tag svg is missing?

Yes, users will see the error that @vomikan is showing `There is 1 missing or invalid icon in chosen theme`. The system will automatically revert to the bundled MMEX Default theme which should always have the correct icons. Users are responsible to update their custom themes if they want to continue using them.

**Comment by n-stein** [comment]
> Hence the question, is it necessary to spend time searching for this leak?

We should probably fix it, but I would consider it a very low priority.

**Comment by tactilis** [comment]
> Yes, users will see the error that @vomikan is showing There is 1 missing or invalid icon in chosen theme. The system will automatically revert to the bundled MMEX Default theme which should always have the correct icons. **Users are responsible to update their custom themes if they want to continue using them.**

Yes, I agree with that.
When the release notes are produced that accompany the introduction of tags, we should remember to add a note about a new TAG.svg icon being needed in custom themes.

**Comment by rmelillo76** [comment]
> 
> But what about any user defined themes? Are they going to throw an error if the tag svg is missing?

Yes it will. I use a custom theme that I based on 'colorful' a while back and I've had to manually copy files into it and re-import on a few occasions.

**Comment by tosim91** [comment]
Re: For Linux builds we are now only officially supporting [Linux Flatpak images](https://flathub.org/apps/org.moneymanagerex.MMEX)   There are many of US Linux users who will not use Flatpack, nor Snap pkgs. Please continue making deb pkgs, and/or, if possible, AppImage, pkgs. Thank you.

**Comment by luisdev** [comment]
Excuse by ignorance... but why won't US Linux users use Flatpack or Snap packages?

**Comment by tosim91** [comment]
That "US" was just to emphasize "us"-we, and not the country. Sorry you
misunderstood.
